usr/share/rear/conf/default.conf

```diff
@@ -503,7 +503,8 @@ BACKUP_PROG_WARN_PARTIAL_TRANSFER=1
 # then you also have to set the CREATE and RESTORE archive options. They are *ignored* if the
 # backup program is supported.
 # default setting for BACKUP_PROG_OPTIONS="" became "--anchored" (GD, 02/DEC/2014 - issue #475)
-BACKUP_PROG_OPTIONS="--anchored"
+# BACKUP_PROG_OPTIONS used to be a string variable, turn it into an array (GD, 06/SEP/2017 - issue #1175)
```

Comment:
This is a breaking change so that we should mention it in the release notes with an explanation of what users have to adjust.

Reply:
@schlomo Yes that is the plan!